src/Microsoft.DotNet.Interactive/Microsoft.DotNet.Interactive.csproj

```diff
@@ -1,16 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework Condition="'$(TargetFramework)'==''">netstandard2.1</TargetFramework>
```

Comment:
Why not multitarget here and remove the `*.Netstandard20` project altogether?  The resultant NuGet package will have both netstandard2.0 and 2.1 binaries.